tests/functional/processes/file/test_fulfillment_process.py

```diff
@@ -0,0 +1,111 @@
+from processes import ClusterProcess, FulfillURLManifestProcess
+from model import Record, Item, Edition, Work, Link
+import json
+import os
+from sqlalchemy.orm import joinedload
+
+
+def test_fulfillment_process(db_manager, s3_manager, seed_test_data):
+
+    manifest_record_uuid = seed_test_data['manifest_record_uuid']
+
+    cluster_process = ClusterProcess('complete', None, None, manifest_record_uuid, None)
```

Comment:
Since we aren't testing the cluster process, you could bypass this and just create a test work, edition, item, link and S3 manifest file.